deployer/templates/fluentd.yaml

```diff
@@ -329,13 +323,9 @@ parameters:
   name: USE_MUX
   value: "false"
 -
-  description: 'Configure MUX CLIENT (false|true).'
-  name: USE_MUX_CLIENT
-  value: "false"
--
-  description: 'Configure MUX SERVER (false|true).'
-  name: MUX_ALLOW_EXTERNAL
-  value: "false"
+  description: 'Configure MUX CLIENT MODE (minimal|full_no_k8s_meta).'
```

Comment:
> If we replace the fluentd collector in the future with something else, would this option still be applicable?

yes, because `full_no_k8s_meta` is not specific to fluentd.

> Do you believe this name is general enough that it would apply in other situations? 

No.  It applies only in the specific situation where you want to offload k8s processing.

So how about `maximal` instead, which implies doing the maximum possible amount of processing?